rack-mount/catalog/README.md

```diff
@@ -4,3 +4,5 @@ A hodgepodge of various pre-configured rack mount parts.
 
 Please feel free to submit a pull request for any new parts you end up creating! Code quality is no object here.
 
+* PLEASE NOTE: NAB9, UM890PRO, EQi12 and 120mm fan tray are NOT compatible with the nano build of the rack stack, they will only fit inside of a Micro or Mini profile.
```

Comment:
For these two notes, I think it would be better to push them down to the individual files. I think it makes sense to include a general note to watch out for size incompatibilities though.

Reply:
I agree, I am removing my change to the readme and I have instead pushed the nano incompatibility to the top comments of each file.